scripts/gitlab/check_runtime.sh

```diff
@@ -47,74 +45,79 @@ then
+	echo "# checking ${1}"
+	# check for spec_version updates: if the spec versions changed, then there is
+	# consensus-critical logic that has changed. the runtime wasm blobs must be
+	# rebuilt.
 
-# check for spec_version updates: if the spec versions changed, then there is
-# consensus-critical logic that has changed. the runtime wasm blobs must be
-# rebuilt.
-
-add_spec_version="$(git diff origin/master...${CI_COMMIT_SHA} ${VERSIONS_FILE} \
-	| sed -n -r "s/^\+[[:space:]]+spec_version: +([0-9]+),$/\1/p")"
-sub_spec_version="$(git diff origin/master...${CI_COMMIT_SHA} ${VERSIONS_FILE} \
-	| sed -n -r "s/^\-[[:space:]]+spec_version: +([0-9]+),$/\1/p")"
+	add_spec_version="$(git diff origin/master...${CI_COMMIT_SHA} ${1} \
+		| sed -n -r "s/^\+[[:space:]]+spec_version: +([0-9]+),$/\1/p")"
+	sub_spec_version="$(git diff origin/master...${CI_COMMIT_SHA} ${1} \
+		| sed -n -r "s/^\-[[:space:]]+spec_version: +([0-9]+),$/\1/p")"
 
 
-# see if the version and the binary blob changed
-if [ "${add_spec_version}" != "${sub_spec_version}" ]
-then
+	# see if the version and the binary blob changed
+	if [ "${add_spec_version}" != "${sub_spec_version}" ]
+	then
 
-	github_label "B2-breaksapi"
+		github_label "B2-breaksapi"
 
-	cat <<-EOT
+		cat <<-EOT
+			
+			changes to the runtime sources and changes in the spec version.
 		
-		changes to the runtime sources and changes in the spec version.
-	
-		spec_version: ${sub_spec_version} -> ${add_spec_version}
-	
-	EOT
-	exit 0
+			spec_version: ${sub_spec_version} -> ${add_spec_version}
+		
+		EOT
+		return
 
-else
-	# check for impl_version updates: if only the impl versions changed, we assume
-	# there is no consensus-critical logic that has changed.
+	else
+		# check for impl_version updates: if only the impl versions changed, we assume
+		# there is no consensus-critical logic that has changed.
 
-	add_impl_version="$(git diff origin/master...${CI_COMMIT_SHA} ${VERSIONS_FILE} \
-		| sed -n -r 's/^\+[[:space:]]+impl_version: +([0-9]+),$/\1/p')"
-	sub_impl_version="$(git diff origin/master...${CI_COMMIT_SHA} ${VERSIONS_FILE} \
-		| sed -n -r 's/^\-[[:space:]]+impl_version: +([0-9]+),$/\1/p')"
+		add_impl_version="$(git diff origin/master...${CI_COMMIT_SHA} ${1} \
+			| sed -n -r 's/^\+[[:space:]]+impl_version: +([0-9]+),$/\1/p')"
+		sub_impl_version="$(git diff origin/master...${CI_COMMIT_SHA} ${1} \
+			| sed -n -r 's/^\-[[:space:]]+impl_version: +([0-9]+),$/\1/p')"
 
 
-	# see if the impl version changed
-	if [ "${add_impl_version}" != "${sub_impl_version}" ]
-	then
-		cat <<-EOT
-		
-		changes to the runtime sources and changes in the impl version.
+		# see if the impl version changed
+		if [ "${add_impl_version}" != "${sub_impl_version}" ]
+		then
+			cat <<-EOT
+			
+			changes to the runtime sources and changes in the impl version.
 
-		impl_version: ${sub_impl_version} -> ${add_impl_version}
+			impl_version: ${sub_impl_version} -> ${add_impl_version}
 
-		EOT
-		exit 0
-	fi
+			EOT
+			exit 0
```

Comment:
LGTM in general, but shouldn't this also be `return`?
made also an [update](https://github.com/paritytech/polkadot/tree/ci-a-runtimes) but your is also good to go with for me.

Reply:
Merged in your commit